src/main/java/de/tum/in/www1/artemis/service/connectors/athena/AthenaSubmissionSendingService.java

```diff
@@ -100,8 +102,8 @@ public void sendSubmissions(Exercise exercise, int maxRetries) {
      * @param submissions the submissions to send
      * @param maxRetries  number of retries before the request will be canceled
      */
-    public void sendSubmissions(Exercise exercise, Set<Submission> submissions, int maxRetries) {
-        Set<Submission> filteredSubmissions = new HashSet<>(submissions);
+    public void sendSubmissions(de.tum.in.www1.artemis.domain.Exercise exercise, Set<de.tum.in.www1.artemis.domain.Submission> submissions, int maxRetries) {
```

Comment:
In my opinion the change from `ExerciseDTOInterface` (and all similar occurrences) leads to code that is more difficult to read and understand: 

- here we have to write out the complete domain due to conflicting namespaces, which leads to very long names.
- In other classes we return objects (e.g. `Exercise`) that are actually a DTO and not an entity, but developers would only see that if they looked at the imports. 

Can you maybe elaborate what your motivation was behind changing this?

Reply:
I agree that it's not ideal, but I would also avoid putting "DTO" and "Interface" into an interface. I am open to suggestions

Reply:
Maybe it makes sense to call the Interface `ExerciseBaseDTO` and then the DTO `ExerciseDTO` so interfaces are always the Base for an DTO.